### PR TITLE
Add more helpful reason strings to TrailingCommaRule

### PR DESCRIPTION
**Pull request by mattrubin**
Also, unwraps some previously-wrapped lines that fit within the [new line length limit](https://github.com/realm/SwiftLint/pull/1010).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1005](https://github.com/realm/SwiftLint/issues/1005)
 
+* Add more helpful reason strings to TrailingCommaRule.  
+  [Matt Rubin](https://github.com/mattrubin)
+  [#1034](https://github.com/realm/SwiftLint/pull/1034)
```
Comment (jpsim) [inline CHANGELOG.md]: we link to the issue a changelog entry resolves, or none at all. See https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

File: Tests/SwiftLintFrameworkTests/TrailingCommaRuleTests.swift [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        )
+    }
+
+    private func trailingCommaViolations(_ string: String, ruleConfiguration: Any? = nil) -> [StyleViolation] {
+        let config = makeConfig(ruleConfiguration, TrailingCommaRule.description.identifier)!
+        return SwiftLintFrameworkTests.violations(string, config: config)
+    }
+}
+
+extension TrailingCommaRuleTests {
```
Comment (marcelofabri) [inline Tests/SwiftLintFrameworkTests/TrailingCommaRuleTests.swift]: you need to add `TrailingCommaRuleTests` to `LinuxMain.swift`

Comment (mattrubin) [inline Tests/SwiftLintFrameworkTests/TrailingCommaRuleTests.swift]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (mattrubin) [inline CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -39,6 +39,9 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1005](https://github.com/realm/SwiftLint/issues/1005)
 
+* Add more helpful reason strings to TrailingCommaRule.
```
Comment (marcelofabri) [inline CHANGELOG.md]: this is now in the wrong section because of SwiftLint 0.15

Comment (mattrubin) [inline CHANGELOG.md]: It's hard to keep up with all these new release! 😄

Comment (marcelofabri) [inline CHANGELOG.md]: Sorry about that 😅

**Pull request merged**